### PR TITLE
Update GetClrDbg.sh to dynamically generate a runtime id

### PR DESCRIPTION
**Pull request by chuckries**
Tested on ubuntu.14.04-x64 and osx.10.10-x64

### CODE REVIEW

**Comment by chuckries** [comment]
@gregg-miskelly

File: scripts/GetClrDbg.sh [hunk 1]
```diff
@@ -95,8 +134,17 @@ fi
 
 # For the rest of this script we can assume the working directory is the install path
 
+echo 'Info: Determinig Runtime ID'
+__RuntimeID=
+get_dotnet_runtime_id
+if [ -z $__RuntimeID ]; then
+    echo "Error: Unable to determine dotnet Runtime ID"
```
Comment (gregg-miskelly) [inline scripts/GetClrDbg.sh]: Should we indicate that a new version of .NET CLI is required?

**Comment by gregg-miskelly** [comment]
Otherwise LGTM

File: scripts/GetClrDbg.sh (hunk 1 above)
Comment (chuckries) [inline scripts/GetClrDbg.sh]: Sure. What is the lowest version of .net cli we need?

Comment (gregg-miskelly) [inline scripts/GetClrDbg.sh]: I believe it would be: 1.0.0-beta-002035

Comment (gregg-miskelly) [inline scripts/GetClrDbg.sh]: Thinking about it more: debugging probably doesn't work in 2035, so I would suggest using 1.0.0-beta-002173

**Pull request closed without merging**